### ISSUE TITLE
Requested region is outside the largest possible region

### ISSUE THREAD

**Issue by resnow92** [olivierhagolle/Start_maja#9]
Hi, 
I am trying to use Maja with start_maja.py
DEM is correctly generated and i followed all instructions as suggested.
but when I try to launch the script start_maja, this error is displayed after 1-2 minutes :  

ScientificProcessing: ITK Exception: Requested region is (at least partially) outside the largest possible region. [vnsGridResampleImageFilter.txx:231]  [vnsMajaMainProcessor.cxx:main:128]

Have you ever seen this error ?
thanks for your help

**Comment by olivierhagolle** [comment]
Hi, we are lacking information to answer this question.

The most probable cause is the fact that DEM and image do not correspond exactly, probably because there was an error on the definition of the DEM size, UL corners. Could you please check, using gdalinfo, that the corners of the 10m resolution MNT are the same as those of the L1C image used as input ?

Best regards,
Olivier

**Comment by olivierhagolle** [comment]
Our contractor, CS-SI confirmed that this message comes from a disagreement between DEM and Image fouutprint.

**Comment by resnow92** [comment]
Thanks, indeed it is a problem with DEM generation. 
The first error concerned the UL corners. I changed them and I re-processed the DEM correctly.

I was confronted to a second problem: some shapefiles for water body information were missing on the USGS Earth Explorer (or inexistant). So i created some empty shapefiles with the name of the tile required. For exemple tile w016n17f was missing and is located in the desert. So I generated an empty shapefile, as there is no water in the region, named w016n17f.shp. 

The DEM is generated and Maja is well processed. 

Thanks for your help.
